testing/internal/e2e/tests/database/migration_test.go

```diff
@@ -195,6 +195,22 @@ func setupEnvironment(t testing.TB, c *config, boundaryRepo, boundaryTag string)
 	})
 	require.NoError(t, err)
 
+	err = pool.Retry(func() error {
+		response, err := http.Get(fmt.Sprintf("%s/health", b.UriLocalhost))
+		if err != nil {
+			t.Logf("Could not access health endpoint: %s. Retrying...", err.Error())
+			return err
+		}
+
+		if response.StatusCode != http.StatusOK {
+			return fmt.Errorf("Health check returned an error. Status Code: %d", response.StatusCode)
+		}
+
+		response.Body.Close()
+		return nil
+	})
+	require.NoError(t, err)
```

Comment:
Should we do this and the non-health one above or just one of them? I feel like we could just remove the first one?

Reply:
Done